azavea/raster-foundry
# Add route for new tool run (run2), tool navbar

## Overview
Add route for new tool run (tool2), tool navbar
Remove old secondary navbar from tool run
Replicate run route as run2 with separate components for further work
Some cleanup

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo
![image](https://user-images.githubusercontent.com/4392704/27703304-e5b2fc32-5cd4-11e7-8f54-54149a6fa2f9.png)

### Notes

The new route is at http://localhost:9091/lab/{toolid}/run2/, and is identical except for a small text change to differentiate while developing 
![image](https://user-images.githubusercontent.com/4392704/27703330-015cf62c-5cd5-11e7-9385-7f79e9061032.png)
None of the dropdown links do anything right now, but that's not a change from the old toolbar
All that's needed to replace the old run view with the new one is renaming everything that's referred to as `run2` to `run` and deleting the stuff it replaces.

## Testing Instructions

* Verify that the new toolbar shows up
* Verify that the new route exists, and changes can be made without affecting the original route

Closes #2155
